docs/src/main/asciidoc/security-enabling-basic-auth-tutorial.adoc

```diff
@@ -0,0 +1,24 @@
+[id="security-enabling-basic-auth-tutorial"]
+= Enabling Basic Authentication
+
+The following tutorial describes how to enable the Basic Authentication for your Quarkus project and allow users to authenticate with a username and password.
```

Comment:
```suggestion
Enable the Basic Authentication for your Quarkus project and allow users to authenticate with a username and password.
```

Reply:
@ebullient We, as the team, decided to write abstracts in a way in which reader will instantly see what template this is. The way this suggestion is written is more suitable as text for the first step of the procedure.

Reply:
Please, see for the context:
https://docs.google.com/document/d/12mIrACRfgs96dfRLWYZM-w1Pmq1cqsMN951k2qdZVOg/edit#heading=h.sqmnacxxb9wb

Reply:
Yes, I'll add my comments to the doc. That doesn't work for how we would like to include/view these documents on longer lists. We have to be able to compromise here a little bit.

Within a view that lists templates along with the preamble, having all Tutorials begin their descriptions with "In this tutorial you will..." or "The following tutorial describes..." is very redundant. It will be clearly listed and navigated as a Tutorial with lots of visual indicators (which can be achieved in docs, too). 

This is an element of writing that comes from longer form, book-like works... I'd like to see if we can streamline/remove some of these filler words and rely on surrounding navigation cues to simplify language.

Reply:
I understand. OK. Lets us chat about it when we will be all back from PTOs.